recipes/stages/_base_/data/seg_semisl.py

```diff
@@ -0,0 +1,30 @@
+__dataset_type = ''
+__data_root = ''
+__pipeline = ''
+
+data = dict(
+    train=dict(
+        type='RepeatDataset',
```

Comment:
Do we need `RepeatDataset` wrapper?

Reply:
I tried to remove but the error came out.. I need to clean and go over the stage.py,(it's gonna be a time-consuming job, removing redundant cfg). Let's leave as is now and remove them later.